franklin/github/views.py

```diff
@@ -23,51 +24,19 @@ def repository_list(request):
     """
     Get all repos currently deployed by Franklin that the user can manage or
     register a new repo
-    ---
-
-    type:
-        200:
-            type: string
-            description: Successful Update
-        201:
-            type: string
-            description: Successful Creation
-
-    response_serializer: SiteSerializer
-    request_serializer: SiteSerializer
-    omit_serializer: false
-
-    parameters_strategy:
-        form: merge
-    parameters:
-        - name: name
-          type: string
-          required: true
-        - name: github_id
-          type: integer
-          required: true
-        - name: owner
-          pytype: builder.serializers.OwnerSerializer
-          required: true
-    responseMessages:
-        - code: 400
-          message: Invalid json received or Bad Request from Github
-        - code: 403
-          message: Current user does not have permission for this repo
-        - code: 422
-          message: Validation error from Github
-        - code: 500
-          message: Error from Github.
     """
-
     if request.method == 'GET':
         if request.user.details.sites.count() == 0:
             github_repos = request.user.details.get_user_repos()
             # TODO - return error from github
             request.user.details.update_repos_for_user(github_repos)
         sites = request.user.details.sites.filter(is_active=True).all()
-        serializer = SiteSerializer(sites, many=True)
-        return Response(serializer.data)
+        site_serializer = SiteSerializer(sites, many=True)
+        user_serializer = UserSerializer(request.user)
+        return Response({
+            'repos': site_serializer.data,
+            'user': user_serializer.data
+        }, status=status.HTTP_200_OK)
```

Comment:
@camiriveron several payloads are now different. Note that the data from the previous response can be found inside `repos`. The `user` object will contain all user data. This will occur on all GET requests that require the passing of an oAuth token. Not currently planning to do this to POST endpoints. We can discuss that though.